backend/src/main/java/de/thu/thutorium/api/transferObjects/CourseCategoryTO.java

```diff
@@ -1,4 +1,4 @@
-package de.thu.thutorium.api.transferObjects.common;
```

Comment:
please pull from main to update the packages. We have common, authentication and chat rn